### DISCUSSION TITLE
Nuxt Image + AWS

### DISCUSSION THREAD

**Discussion by danvega** [nuxt/image#791] [General]
I built a blog that uses Notion as the CMS. The author writes posts in Notion and uploads images there. When I am reading in the data the image comes from an S3 bucket. I was looking into using Nuxt Image so that I could improve performance in the following areas: 

- Serve images in next-gen formats
- Properly size images
- specify width and hight of image

Another odd issue with Notion images is they expire in 60 minutes so I can't cache them for long. Am I able to use Nuxt Image in this scenario? I guess I'm just a little bit confused on what nuxt image is doing for remote sources. 

Thank you!

**Comment by DavidDeSloovere** [comment]
I don't see AWS/S3 as a provider. Also this component is for manipulating images "on the fly" (cached though). So I don't think your use-case fits here.

**Comment by oemer-aran** [comment]
I guess what we really need is a solution from AWS using S3 + Cloudfront. You could use Providers like Cloudinary and their CDN, but why if we already have AWS?
